dist/origin-web-common-services.js

```diff
@@ -2893,7 +2908,37 @@ angular.module('openshiftCommonServices')
       // Returns a promise that resolves with {token:'...',then:'...',verified:true|false}, or rejects with {error:'...'[,error_description:'...',error_uri:'...']}
       // If no token and no error is present, resolves with {}
       // Example error codes: https://tools.ietf.org/html/rfc6749#section-5.2
-      finish: function() {
+      finish: function($http) {
```

Comment:
agree it's gorpy to require this here... @spadgett was looking into avoiding this